Event.java

```diff
@@ -1,12 +1,12 @@
-public class Event {
-
-    private Date date;
-    private String location;
-    private TimeSlot time;
-    private String Contact;
-
-public static void main(String [] args){
-    new
-}
-
-}
+//public class Event {
```

Comment:
Probably shouldn't make a change to every class in this pr sort of out of the scope. Why not just make changes to Date? Most of the changes in classes other than Date are just comments we don't really need.

Reply:
Roger that